### PR TITLE
[stable/neo4j] Add support for imagePullSecrets

### PR DESCRIPTION
**Pull request by olemarkus**
@mneedham 

**What this PR does / why we need it**:
Adds support for registry secrets

### CODE REVIEW

File: stable/neo4j/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: neo4j
 home: https://www.neo4j.com
-version: 0.5.1
+version: 0.5.2
```
Comment (timstoop) [inline stable/neo4j/Chart.yaml]: This adds a new feature, so you can bump the minor version.

**Review by timstoop — CHANGES_REQUESTED** [review]
Looks fine to me and a good feature to add.

**Review by timstoop — APPROVED** [review]
Thanks! Looks fine now.

File: stable/neo4j/values.yaml [hunk 2]
```diff
@@ -9,6 +9,7 @@ name: "neo4j"
 image: "neo4j"
 imageTag: "3.2.3-enterprise"
 imagePullPolicy: "IfNotPresent"
+# ImagePullSecret: registry-secret
```
Comment (unguiculus) [inline stable/neo4j/values.yaml]: lower case: `imagePullSecret`

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/neo4j/values.yaml (hunk 2 above)
Comment (olemarkus) [reply stable/neo4j/values.yaml]: Good catch

**Review by unguiculus — APPROVED** [review]

**Pull request merged**